<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Browser</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDownload">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSaveMeta">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Metadata</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAddLayer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add as a layer</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbTitle">
      <property name="text">
       <string>TITLE:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbTitleText">
      <property name="text">
       <string>Title Text</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayoutMetadata">
      <item row="0" column="0">
       <widget class="QLabel" name="lb0">
        <property name="text">
         <string>PLATFORM:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbText0">
        <property name="text">
         <string>lbText0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb1">
        <property name="text">
         <string>ACQUISITION START:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbText1">
        <property name="text">
         <string>lbText1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb2">
        <property name="text">
         <string>ACQUISITION END:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbText2">
        <property name="text">
         <string>lbText2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb3">
        <property name="text">
         <string>GSD:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbText3">
        <property name="text">
         <string>lbText3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb4">
        <property name="text">
         <string>PROVIDER:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lbText4">
        <property name="text">
         <string>lbText4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb5">
        <property name="text">
         <string>FILE SIZE:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lbText5">
        <property name="text">
         <string>lbText5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
